app/controllers/articles_controller.rb

```diff
@@ -1,4 +1,49 @@
 class ArticlesController < ApplicationController
   def index
+    @articles = Article.all
+  end
+
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+
+    if @article.save
+      redirect_to @article # use redirect_to when mutating app state, rather than render...
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    @article = Article.find(params[:id])
+  end
+
+  def edit
+    @article = Article.find(params[:id])
+  end
```

Comment:
Is `ArticlesController#edit` simply meant to be a synonym of `#show`? If so then does the following work?
```rb
  def show
     @article = Article.find(params[:id])
  end
  alias_method :edit, :show
```